varia/sass/abstracts/_mixins.scss

```diff
@@ -97,6 +97,5 @@
  * It also adds the proper fallback for browsers without support.
  */
 @mixin font-family( $config_map ) {
-	font-family: #{map-deep-get( $config_map, 'fallback' )}; // For browsers without CSS custom properties support.
```

Comment:
The postcss-custom-properties plugin will take care of generating the fallback rules for us. One thing I'd like to do is refactoring how this works and remove the mixin. This is perhaps best accomplished in a subsequent PR as to keep the focus on line-heights in this one.

Reply:
Interesting. Originally, I made Varia (previously called `_dsgnsystm`) with _only_ CSS-variables (see [here](https://github.com/allancole/_dsgnsystm/tree/master/_dsgnsystm-theme)). 

Are you basically suggesting we go back to that? I’d definitely prefer that direction 👍

Reply:
Wait, I just answered my own question. Nevermind ;-)

Reply:
Go back to the roots! Yeah, I think this is clearer than the mixin approach. Although I still need to figure out how the overrides would work (global configs VS blocks/components configs), it is promising.